### PR TITLE
Add WP_CLI command interface.

### PR DESCRIPTION
**Pull request by strangerkir**
This PR is made to add the WP_CLI command interface. Ideas for the interface is taken from [this comment](https://github.com/inpsyde/idealo-cli-command/pull/1#discussion_r458646181).

I also added tests in the same way it was done for the existing interfaces. I have doubts if they are useful in this case, but I did it at least for consistency.

### CODE REVIEW

File: src/WpCli/WpCliCommandInterface.php [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    public function getWhen(): ?string;
+
+    /**
+     * Whether command registering was deferred.
+     *
+     * @return bool|null Whether command deferred.
+     */
+    public function isDeferred(): ?bool;
+
+}
```
Comment (XedinUnknown) [inline src/WpCli/WpCliCommandInterface.php]: EOL missing

File: src/WpCli/WpCliCommandInterface.php [hunk 2]
```diff
@@ -0,0 +1,75 @@
+<?php
+
+declare (strict_types = 1);
+
+namespace WpOop\WordPress\WpCli;
+
+/**
+ * Represents WP_CLI command.
```
Comment (XedinUnknown) [inline src/WpCli/WpCliCommandInterface.php]:
```suggestion
 * Represents a WP_CLI command.
```

File: tests/unit/WpCli/WpCliCommandInterfaceTest.php [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+        return $subject;
+    }
+
+    public function testInstantiation(): void
+    {
+        $subject = $this->createSubject();
+
+        $this->assertInstanceOf(Subject::class, $subject);
+    }
+}
```
Comment (XedinUnknown) [inline tests/unit/WpCli/WpCliCommandInterfaceTest.php]: EOL

**Review by XedinUnknown — CHANGES_REQUESTED** [review]

File: src/WpCli/WpCliCommandInterface.php (hunk 1 above)
Comment (strangerkir) [reply src/WpCli/WpCliCommandInterface.php]: Fixed.

File: tests/unit/WpCli/WpCliCommandInterfaceTest.php (hunk 3 above)
Comment (strangerkir) [reply tests/unit/WpCli/WpCliCommandInterfaceTest.php]: Fixed.

**Pull request closed without merging**